app/Models/Signup.php

```diff
@@ -197,29 +197,13 @@ public function toQuasarPayload()
     }
 
     /**
-     * Get either the quantity or quantity_pending for a signup.
-     * If the quantity lives on the signup's posts,
-     * return quantity as a summed total of quantity across all posts under a signup.
+     * Get the total quantity for this signup's posts.
      *
      * @return int
      */
     public function getQuantity()
     {
-        // @TODO - This is temporary. We have migrated data that has stored quanity in the
-        // quanity_pending column on the signup. However, since then we updated the business
-        // logic to store everything in the quanity column and not use the quanity_pending
-        // column at all. We only want to return what is in the quanity_pending column
-        // if is the only place quanity is stored.
-        if (! config('features.v3QuantitySupport') || $this->posts->isEmpty()) {
-            if (! is_null($this->quantity_pending) && is_null($this->quantity)) {
-                return $this->quantity_pending;
-            }
-
-            return $this->quantity;
-        }
```

Comment:
I was just [asking about trashing all of this](https://dosomething.slack.com/archives/C0YGXUE01/p1575329708161500) as I was looking into aggregating quantity over in #964.

Reply:
Ah, a good coincidence! 💥